landing/Header/index.js

```diff
@@ -21,6 +21,11 @@ function Header() {
           <li className="cursor-pointer hover:text-primary">Currículo</li>
           <li className="cursor-pointer hover:text-primary">Projetos</li>
           <li className="cursor-pointer hover:text-primary">Equipa</li>
+          <li className="cursor-pointer hover:text-primary">
+            <a href="recruitment">
```

Comment:
Instead of anchor tag, use Link component from `next/link` for navigation between pages.